controllers/runnermanager.go

```diff
@@ -36,16 +39,21 @@ type RunnerManagerImpl struct {
 	githubClient    github.Client
 	runnerPodClient rc.Client
 
-	loops map[string]*managerLoop
+	lastCheckTime *atomic.Value
+	loops         map[string]*managerLoop
 }
 
 func NewRunnerManager(log logr.Logger, interval time.Duration, k8sClient client.Client, githubClient github.Client, runnerPodClient rc.Client) RunnerManager {
+	var lastCheckTime atomic.Value
+	lastCheckTime.Store(time.Now().UTC())
+
```

Comment:
Why this variable is defined here? And why do you use `atomic.Value`?

Reply:
Fixed in d799cb4.